OsvvmScriptsCreateYamlReports.tcl

```diff
@@ -313,6 +393,15 @@ proc FinishTestSuiteBuildYaml {} {
   variable TestSuiteStartTimeMs
 
   set   RunFile  [open ${::osvvm::OsvvmTempYamlFile} a]
+  if {[info exists ::osvvm::TestSuiteTitle] && $::osvvm::TestSuiteTitle ne ""} {
+    WriteDictOfString2Yaml $RunFile Title $::osvvm::TestSuiteTitle "    "
+  }
+  if {[info exists ::osvvm::TestSuiteBrief] && $::osvvm::TestSuiteBrief ne ""} {
+    WriteDictOfString2Yaml $RunFile Brief $::osvvm::TestSuiteBrief "    "
+  }
+  if {[info exists ::osvvm::TestSuiteDescription] && $::osvvm::TestSuiteDescription ne ""} {
+    WriteDictOfString2Yaml $RunFile Description $::osvvm::TestSuiteDescription "    "
+  }
```

Comment:
Care should be taken to make sure that TestSuiteTitle, TestSuiteBrief, and TestSuiteDescription are initialized to "", generally in OsvvmSettingsDefault.tcl and then reset to "" after the test suite ends (end of build or start of new test suite).   That way there is no need to do the exists check.